include/dlaf/util_matrix.h

```diff
@@ -243,7 +243,7 @@ void set_random(Matrix<T, Device::CPU>& matrix) {
       GlobalTileIndex tile_wrt_global = dist.globalTileIndex(tile_wrt_local);
 
       auto tl_index = dist.globalElementIndex(tile_wrt_global, {0, 0});
-      auto seed = tl_index.row() * matrix.size().cols() + tl_index.col();
+      std::size_t seed = to_sizet(tl_index.row() * matrix.size().cols() + tl_index.col());
```

Comment:
What about using size_t operations in this case?
(Commented similar cases with (*))